Zotero/Scenes/Detail/PDF/Views/PDFReaderViewController.swift

```diff
@@ -764,83 +764,84 @@ extension PDFReaderViewController: PDFDocumentDelegate {
         to newVariant: PSPDFKit.Annotation.Variant?
     ) {
         if let state = oldState?.toolbarTool {
-            self.annotationToolbarController.set(selected: false, to: state, color: nil)
+            annotationToolbarController.set(selected: false, to: state, color: nil)
         }
 
         if let state = newState {
-            let color = self.viewModel.state.toolColors[state]
+            let color = viewModel.state.toolColors[state]
             if let tool = state.toolbarTool {
-                self.annotationToolbarController.set(selected: true, to: tool, color: color)
+                annotationToolbarController.set(selected: true, to: tool, color: color)
             }
         }
     }
 
     func didChange(undoState undoEnabled: Bool, redoState redoEnabled: Bool) {
-        self.annotationToolbarController.didChange(undoState: undoEnabled, redoState: redoEnabled)
+        annotationToolbarController.didChange(undoState: undoEnabled, redoState: redoEnabled)
     }
 
     func interfaceVisibilityDidChange(to isHidden: Bool) {
-        let shouldChangeNavigationBarVisibility = !self.toolbarState.visible || self.toolbarState.position != .pinned
+        let shouldChangeNavigationBarVisibility = !toolbarState.visible || toolbarState.position != .pinned
 
-        if !isHidden && shouldChangeNavigationBarVisibility && self.navigationController?.navigationBar.isHidden == true {
-            self.navigationController?.setNavigationBarHidden(false, animated: false)
-            self.navigationController?.navigationBar.alpha = 0
+        if !isHidden, shouldChangeNavigationBarVisibility, navigationController?.navigationBar.isHidden == true {
```

Comment:
Why change from `&&` to `,`? I'd vote for keeping normal logic operators (`&&`, `||`) and use `,` only when actually needed.

Reply:
For clauses with only and operators, this seems easier to read for me, but I understand there might be cases where a logic operator is preferred. I suggest that we always use `,` for `guard` statements, and for `if` statements that contain `let` clauses. WDYT?

Reply:
No strong feelings on my side, so to be on the same side, only let clauses, may be separated by `,` and there is no need to re-style existing cases using `&&`.

Reply:
Check https://github.com/zotero/zotero-ios/compare/c00971089803a374cdf9edf03132eeab0663457a..4293218569cd0a3852646beda65ed98e02353d22